geopandas/tests/util.py

```diff
@@ -37,6 +37,34 @@ def validate_boro_df(df, case_sensitive=False):
     assert Series(df.geometry.type).dropna().eq("MultiPolygon").all()
 
 
+def connect_engine(dbname, user=None, password=None, host=None, port=None):
+    """
```

Comment:
This is duplicating the version in test_sql.py?

Reply:
Yeah indeed seemed to do so. I removed the one from `test_sql.py` and use the one from `util.py` now (also fixed it as it missed some imports).

Reply:
Aaand let's take it back, just saw your comment about using the fixture. 👍

Reply:
@jorisvandenbossche I guess we can now clean all the functions from `util.py` related to creating the connections?

Reply:
Yes, normally there shouldn't be any functions left in here that create a connection (the `create_..` functions below get a connection passed)